libsolidity/ast/Types.cpp

```diff
@@ -1877,23 +1877,22 @@ TypeResult ArrayType::interfaceType(bool _inLibrary) const
 	if (!_inLibrary && m_interfaceType.is_initialized())
 		return *m_interfaceType;
 
-	TypePointer result;
+	TypeResult result{TypePointer{}};
+	TypeResult baseExt = m_baseType->interfaceType(_inLibrary);
```

Comment:
What does `Ext` refer to?

Reply:
I just re-orded & adjusted the existing code which already had & used `baseExt`, so.. I actually have no idea. Something related to `External` maybe? because.. external type?

Reply:
I think the function was called `externalType` before.